SETUP/tests/ci_configuration.sh

```diff
@@ -47,6 +47,8 @@ _API_ENABLED=true
 _API_RATE_LIMIT=false
 _API_RATE_LIMIT_REQUESTS_PER_WINDOW=3600
 _API_RATE_LIMIT_SECONDS_IN_WINDOW=3600
+_API_RATE_LIMIT_SECONDS_IN_WINDOW=3600
+_API_CLIENT_STORAGE_KEYS='[]'
```

Comment:
```suggestion
_API_CLIENT_STORAGE_KEYS='[]'
```
@srjfoo caught this - copy/pasta error on my part and I'll get it fixed.